mycroft/skills/skill_updater.py

```diff
@@ -219,6 +219,15 @@ def install_or_update(self, skill):
                 raise
         self.installed_skills.add(skill.name)
 
+    def defaults_installed(self):
+        """Check if all default skills are installed.
+
+        Returns:
+            True if all default skills are installed, else False.
+        """
+        defaults = self.msm.list_all_defaults()[self.msm.platform]
+        return all([skill.is_local for skill in defaults])
```

Comment:
Do we need to account for people blacklisting default Skills?

Reply:
Hmm, blacklisting skills still installs them, it just doesn't load them afaik. It's an update we can do but we should probably change the update method as well.

Reply:
I've pushed an update to exclude blacklisted skills from the default check. I've also looked at excluding blacklisted skills from the install/update procedure which is fairly easy to add, if we think that should be added as well?

Reply:
Working great. 

Yeah I think that makes sense too. If someone has blacklisted a Skill we don't need to install it. Though if there is some security update, perhaps we do want to update? 

I see that update and install are handled separately so we could do updates but not installs? Really though, this is a nice to have so I'm going to merge this PR as is and we can do the above in the future.